src/coreclr/vm/callconvbuilder.cpp

```diff
@@ -60,7 +60,7 @@ namespace
     {
 #define BASE_CALL_CONV(name, flag) { \
         MAKE_FULLY_QUALIFIED_CALLCONV_TYPE_NAME_PREFIX(name), \
-        STRING_LENGTH(MAKE_FULLY_QUALIFIED_CALLCONV_TYPE_NAME_PREFIX(name)), \
+        STRING_LENGTH(MAKE_FULLY_QUALIFIED_CALLCONV_TYPE_NAME_PREFIX(name)) - 1, \
```

Comment:
Should it be instead:
```suggestion
        ARRAY_SIZE(MAKE_FULLY_QUALIFIED_CALLCONV_TYPE_NAME_PREFIX(name)) - 1, \
```
(or -2 with `STRING_LENGTH`). It is defined here:
https://github.com/dotnet/runtime/blob/0df121458740db1d0dd6cbe2c24bd0c94fb0af5e/src/native/minipal/utils.h#L7-L10
Previously, it was `lengthof(MAKE_FULLY_QUALIFIED_CALLCONV_TYPE_NAME_PREFIX(name)) - 1, \` where `lengthof` was defined as `#define lengthof(rg)    (sizeof(rg)/sizeof(rg[0]))`.

Reply:
I'm missing something here. As noted above the `lengthof` approach was previously identical to `ARRAY_SIZE`. The update in the referenced PR was then converted to `STRING_LENGTH`, which is `ARRAY_SIZE - 1`. The intent here, if I recall, is to get the string character count, minus the `null`. That seems to be `STRING_LENGTH`? Subtracting another 1, means we are not removing characters from the string. I've no doubt there could be an issue here, but I am missing how this change fixes a problem.

Reply:
You are correct, Disregard my comment. :)

`lengthof - 1` to `STRING_LENGTH` change was correct going by the definition of `lengthof` which was deleted in https://github.com/dotnet/runtime/pull/61537.